### PR TITLE
Fate#318832 Redesign iSCSI Client/Server UI - Authentication.

### PR DESCRIPTION
**Pull request by ls-zhu**
Fate#318832 Redesign iSCSI Client/Server UI - Authentication.

Add two functions: EnableDiscoveryAuth and DisableDiscoveryAuth
A bug fix for discovery session without authentication
Changed online help docs

### CODE REVIEW

**Comment by jreidinger** [comment]
looks good now, just increase version and add entry to changelog

**Comment by ls-zhu** [comment]
A note, the CI failed case in the first commit 71efc0e is ALREADY fixed in the second commit 7382c96

File: package/yast2-iscsi-lio-server.changes [hunk 1]
```diff
@@ -1,6 +1,14 @@
 -------------------------------------------------------------------
 Wed Jul  6 11:00:53 CEST 2016 - gs@suse.de
 
+- fate#318832, Redesign authentication UI
+- A bug fix for discovery session without authentication.
+- Changed online help.
+- 3.1.20
```
Comment (jreidinger) [inline package/yast2-iscsi-lio-server.changes]: this is wrong way how to add changes, please use `osc vc` it adds proper timestamp and also you as author of change.

File: package/yast2-iscsi-lio-server.spec [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-iscsi-lio-server
-Version:        3.1.19
+Version:        3.1.20
```
Comment (jreidinger) [inline package/yast2-iscsi-lio-server.spec]: as this go to master, it should bump version to 3.2.0 which is minor used in SP3

Comment (ls-zhu) [inline package/yast2-iscsi-lio-server.spec]: sure

File: package/yast2-iscsi-lio-server.changes [hunk 3]
```diff
@@ -4,7 +4,7 @@ Wed Feb 22 09:55:05 UTC 2017 - lszhu@suse.com
 - fate#318832, redesign authentication UI
 - changed online help
 - a bug fix for discovery session without authentication.
--3.1.20
+-3.20
```
Comment (jreidinger) [inline package/yast2-iscsi-lio-server.changes]: it should be `3.2.0` not `3.20`

**Pull request merged**